pkg/outputs/nats_outputs/jetstream/jetstream_output.go

```diff
@@ -87,6 +87,7 @@ type config struct {
 	NumWorkers         int                 `mapstructure:"num-workers,omitempty" json:"num-workers,omitempty"`
 	WriteTimeout       time.Duration       `mapstructure:"write-timeout,omitempty" json:"write-timeout,omitempty"`
 	Debug              bool                `mapstructure:"debug,omitempty" json:"debug,omitempty"`
+	BufferSize         int                 `mapstructure:"buffer-size,omitempty"`
```

Comment:
Can we change the type to uint? Avoids having to deal with negative numbers and changing types when using it.

Reply:
This has been updated as requested.

Reply:
This has been updated and fixed as requested.